ember-toucan-styles/package.json

```diff
@@ -40,23 +40,21 @@
   ],
   "scripts": {
     "start": "pnpm watch:js",
-    "build": "concurrently 'npm:build:*'",
-    "build:js": "rollup -c ./rollup.config.js",
-    "build:docs": "cp ../README.md ./README.md",
+    "build": "rollup -c ./rollup.config.mjs",
     "watch:js": "rollup -c --watch --no-watch.clearScreen",
-    "lint": "concurrently 'npm:lint:js'",
-    "lint:fix": "concurrently 'npm:lint:js:fix'",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
```

Comment:
what does the `--names` flag do? js and prettier?

Reply:
yeah, it makes the output nicer:

![image](https://user-images.githubusercontent.com/199018/217652617-509099cb-86c9-47e7-8691-bc2cc874bc39.png)

without, it looks like this: 
```
❯ pnpm concurrently 'npm:lint:*(!fix)'
[lint:*(!fix)hbs]
[lint:*(!fix)hbs] > test-app@0.0.0 lint:hbs
[lint:*(!fix)hbs] > ember-template-lint . --no-error-on-unmatched-pattern
[lint:*(!fix)hbs]
[lint:*(!fix)prettier]
[lint:*(!fix)prettier] > test-app@0.0.0 lint:prettier
[lint:*(!fix)prettier] > prettier -c .
[lint:*(!fix)prettier]
[lint:*(!fix)js]
[lint:*(!fix)js] > test-app@0.0.0 lint:js
[lint:*(!fix)js] > eslint . --cache
[lint:*(!fix)js]
[lint:*(!fix)prettier] Checking formatting...
[lint:*(!fix)hbs] npm run lint:hbs exited with code 0
[lint:*(!fix)prettier] [warn] app/templates/application.hbs
[lint:*(!fix)prettier] [warn] tests/index.html
[lint:*(!fix)prettier] [warn] Code style issues found in 2 files. Forgot to run Prettier?
[lint:*(!fix)prettier] npm run lint:prettier exited with code 1
[lint:*(!fix)js]
[lint:*(!fix)js] <repo>/test-app/app/components/theme-toggler.gjs
[lint:*(!fix)js]   6:2  error  Parsing error: Unexpected token (6:2)
[lint:*(!fix)js]
[lint:*(!fix)js] <repo>/test-app/config/tailwind/tailwind.config.js
[lint:*(!fix)js]    1:1   error  Definition for rule 'node/no-missing-require' was not found  node/no-missing-require
[lint:*(!fix)js]   15:21  error  "@crowdstrike/tailwind-toucan-base" is not found             n/no-missing-require
[lint:*(!fix)js]
[lint:*(!fix)js] <repo>/test-app/types/ember-data/types/registries/model.d.ts
[lint:*(!fix)js]   5:18  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
[lint:*(!fix)js]
[lint:*(!fix)js] ✖ 4 problems (3 errors, 1 warning)
[lint:*(!fix)js]
[lint:*(!fix)js] npm run lint:js exited with code 1
```

and --names fix: looks like this: 
```
❯ pnpm lint:fix

> test-app@0.0.0 lint:fix <repo>/test-app
> concurrently 'npm:lint:*:fix' --names 'fix:'

[fix:prettier]
[fix:prettier] > test-app@0.0.0 lint:prettier:fix
[fix:prettier] > prettier -w .
[fix:prettier]
[fix:js]
[fix:js] > test-app@0.0.0 lint:js:fix
[fix:js] > eslint . --fix
[fix:js]
[fix:hbs]
[fix:hbs] > test-app@0.0.0 lint:hbs:fix
[fix:hbs] > ember-template-lint . --fix --no-error-on-unmatched-pattern
[fix:hbs]
[fix:prettier] .eslintrc.js 28ms
[fix:prettier] .prettierrc.js 4ms
[fix:prettier] .template-lintrc.js 2ms
[fix:prettier] .watchmanconfig 2ms
[fix:hbs] npm run lint:hbs:fix exited with code 0
[fix:prettier] app/app.ts 104ms
[fix:prettier] app/config/environment.d.ts 7ms
[fix:prettier] app/index.html 23ms
[fix:prettier] app/router.ts 4ms
```